mmtf-serialization/src/main/java/org/rcsb/mmtf/dataholders/MmtfStructureFactory.java

```diff
@@ -0,0 +1,128 @@
+package org.rcsb.mmtf.dataholders;
+
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.List;
+import org.rcsb.mmtf.serialization.quickmessagepackdeserialization.ObjectTree;
+
+/**
+ *
+ * @author Antonin Pavelka
+ */
+public class MmtfStructureFactory {

```

Comment:
Could you add some javadocs to explain what this class does? The individual methods are more self-explanatory